### ISSUE TITLE
Formatter is deleting subsequent lines when it finds <</element>

### ISSUE THREAD

**Issue by marcolarosa** [chrisbottin/xml-formatter#52]
If we have a document like:
```
document = `<surface>
   <p>a paragraph<</p>   <-- this closing tag is wrong
   <p>another para</p>
   <p>another para</p>
   <p>another para</p>
</surface>`;
```

And we format it `( format(document, { indentation: "  ", collapseContent: true })) )` we lose everything after the line with the tag `<</p>` (line 2). Output from format:
```
<surface>
  <p>a paragraph</p>
</surface>
```

**Comment by chrisbottin** [comment]
Hi,

The parser should throw an error when the XML is not well formed so the formatting doesn't happen.

I see for this use case the parser fails to detect the bad XML content. I will have a look for a fix

**Comment by chrisbottin** [comment]
Hi @marcolarosa,

This issue should be fixed in the version 3.2.0 (https://www.npmjs.com/package/xml-formatter/v/3.2.0).

In this version, the code will throw an error when you are trying to format some invalid XML.

There is also an option `throwOnFailure` to avoid throwing an error and instead the code will just return the original XML.

I hope that works for you.

**Comment by marcolarosa** [comment]
Yep - thanks. Closing this ticket.
